bitnami/etcd/templates/scripts-configmap.yaml

```diff
@@ -41,7 +41,10 @@ data:
     ETCDCTL_ENDPOINTS="{{range $i, $e := until $replicaCount }}{{ $etcdClientProtocol }}://{{ $etcdFullname }}-{{ $e }}.{{ $etcdHeadlessServiceName }}.{{ $releaseNamespace }}.svc.{{ $clusterDomain }}:{{ $clientPort }},{{ end }}"
     # Remove the last comma "," introduced in the string
     export ETCDCTL_ENDPOINTS="$(sed 's/,/ /g' <<< $ETCDCTL_ENDPOINTS | awk '{$1=$1};1' | sed 's/ /,/g')"
-
+    export ROOT_PASSWORD="${ETCD_ROOT_PASSWORD:-}"
+    if [[ -n "${ETCD_ROOT_PASSWORD:-}" ]]; then
+      unset ETCD_ROOT_PASSWORD
+    fi
```

Comment:
What about renaming this env var here to `ROOT_PASSWORD`: https://github.com/bitnami/charts/blob/master/bitnami/etcd/templates/statefulset.yaml#L136

Reply:
Essentially for having consistency between the expected variable names here https://github.com/bitnami/bitnami-docker-etcd and in the chart. Even if we changed it everywhere, we would need some sort of backwards compatibility